enter/state.c

```diff
@@ -72,6 +72,7 @@ struct EnterState *enter_state_new(void)
   struct EnterState *es = mutt_mem_calloc(1, sizeof(struct EnterState));
 
   enter_state_resize(es, 1);
+  memset(es->wbuf, 0, es->wbuflen);
```

Comment:
Wouldn't it be best if `enter_state_resize` filled the range `[wbuflen,  num)` with zeroes?

Reply:
Yes, but that would require some careful thought.
Better still would be `mutt_realloc_zero()`; I just need some newbie to volunteer.